website/docs/gs/terra.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Terra
+description: Run GATK-SV on your cohort on the Terra platform.
+sidebar_position: 5
+---
+
+# Terra Workspace
```

Comment:
Let's stick with what the README has about Terra for now. The cohort mode workspace is not public yet and should not be referred to on the website.